docker/index.js

```diff
@@ -296,6 +302,7 @@ const processAndUpload = async (
 				code: codeAuditor,
 				htmlIssuesSummary,
 				htmlIssues,
+				isPrivate: args.private ? true : false
```

Comment:
args.private should already be a boolean, don't need this

Reply:
Done - Updated